app/sidekiq/evss/disability_compensation_form/upload_bdd_instructions.rb

```diff
@@ -89,21 +114,20 @@ def perform(submission_id)
       private
 
       def upload_bdd_instructions
-        client.upload(file_body, document_data)
+        if Flipper.enabled?(:disability_compensation_use_api_provider_for_bdd_instructions)
+          provider = self.class.api_upload_provider(submission)
+
+          upload_document = provider.generate_upload_document(BDD_INSTRUCTIONS_FILE_NAME)
+          provider.submit_upload_document(upload_document, file_body)
+        else
+          EVSS::DocumentsService.new(submission.auth_headers).upload(file_body, document_data)
+        end
       end
 
       def file_body
         @file_body ||= File.read('lib/evss/disability_compensation_form/bdd_instructions.pdf')
       end
 
-      def client
-        @client ||= if Flipper.enabled?(:disability_compensation_lighthouse_document_service_provider)
```

Comment:
This was a placeholder switch added over a year ago when another developer was setting up to do this project; I've refactored the existing EVSS upload behavior in this job to move it onto that one line on line 123